jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/main/java/org/jbpm/workbench/pr/backend/server/RemoteProcessRuntimeDataServiceImpl.java

```diff
@@ -72,6 +73,21 @@ public ProcessInstanceSummary getProcessInstance(ProcessInstanceKey processInsta
         return new ProcessInstanceSummaryMapper(processInstanceKey.getServerTemplateId()).apply(processInstance);
     }
 
+    private List<ProcessInstanceSummary> getProcessInstancesByParentId(ProcessInstanceKey processInstanceKey, String containerId) {
+        if (processInstanceKey == null || !processInstanceKey.isValid()) {
+            return null;
+        }
+
+        ProcessServicesClient processServicesClient = getClient(processInstanceKey.getServerTemplateId(), ProcessServicesClient.class);
+        List<ProcessInstance> processInstances = processServicesClient.findProcessInstancesByParent(containerId, processInstanceKey.getProcessInstanceId(),
+                                                                                                    Arrays.asList(org.kie.api.runtime.process.ProcessInstance.STATE_ACTIVE,
+                                                                                                                  org.kie.api.runtime.process.ProcessInstance.STATE_COMPLETED,
+                                                                                                                  org.kie.api.runtime.process.ProcessInstance.STATE_COMPLETED), 0, Integer.MAX_VALUE);
```

Comment:
STATE_COMPLETED is twice here.

Reply:
@bxf12315 I think one of the STATE_COMPLETED have to be STATE_ABORTED